draft code editors extension guide - issue #1549 (#1584)
ember-learn/guides-source: guides/release/code-editors/index.md

> During development, add Ember.js specifc extensions to your code editor for added functionality. Below is a list of some of the extensions available:  

Reviewer: ```suggestion
During development, add Ember.js specific extensions to your code editor for added functionality. Below is a list of some of the extensions available, many of which are created and maintained by the developer community:  
```
Just adding a little bit of messaging to give credit to the authors and to avoid the assumptions that the core team maintains them.

## Visual Studio Code

> Visual Studio Code is a code editor optimized for building and debugging modern web applications. VSC is one of the most popular text editors among ember developers.

Reviewer: ```suggestion
Visual Studio Code is a code editor optimized for building and debugging modern web applications. Visual Studio Code is one of the most popular text editors among Ember developers.
```

> [Unstable Ember Language Server](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable) - This provides autocompletions in templates and allows go-to-definition behavior within ember projects where tsserver wouldn't be able to help.

Reviewer: ```suggestion
[Unstable Ember Language Server](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable) - This provides autocompletions in templates and allows go-to-definition behavior within Ember projects.
```

Can we also mention the Stable Language Server?

[Glimmer Templates Syntax](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-glimmer-syntax) - Syntax formatting for glimmer templates.

> [Related Files Hopper](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) - This extension allows movement within a collection of related files with a single keystroke making development faster and efficient. This is especially useful for finding related files within embers components. 

Reviewer: ```suggestion
[Related Files Hopper](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) - This extension allows movement within a collection of related files with a single keystroke making development faster and efficient. This is especially useful for finding related files within components. 
```

[EditorConfig for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) - Attempts to override user/workspace settings with settings found in .editorconfig files. The .editorconfig file helps developers define and maintain consistent coding styles between different editors and IDEs.

[Prettier for Handlebars](https://marketplace.visualstudio.com/items?itemName=Alonski.prettier-for-handlebars-vscode) - Format your handlebars files with Prettier ...that's it! Note, because this uses the currently unreleased Prettier, it may break with future changes to Prettier.

## vim

Vim is a highly configurable text editor built to make creating and changing any kind of text very efficient. It is included as "vi" with most UNIX systems and with Apple OS X. 

> [ember.vim](https://github.com/dsawardekar/ember.vim) - Ember.vim is a Vim plugin for the Ember.js frontend framework inspired by vim-rails.

Reviewer: ```suggestion
[ember.vim](https://github.com/dsawardekar/ember.vim) -  a Vim plugin for the Ember.js frontend framework inspired by `vim-rails`.
```
Things in backticks don't get spellchecked. Or, we can add this to the `.dic` file in the Guides if we don't want to do backticks here.

> [vim-ember-hbs](https://github.com/joukevandermaas/vim-ember-hbs) - a plugin to add Ember Handlebars/HTMLBars syntax highlighting and indentation to Vim.

Reviewer: ```suggestion
[vim-ember-hbs](https://github.com/joukevandermaas/vim-ember-hbs) - a plugin to add Ember template syntax highlighting and indentation to Vim.
```

> [embre tools](https://github.com/AndrewRadev/ember_tools.vim) - Varrious tools to work with Ember.js projects.

Reviewer: ```suggestion
[embre tools](https://github.com/AndrewRadev/ember_tools.vim) - Various tools to work with Ember.js projects.
```

## Atom

Atom is hackable text editor for the 21st Century.

> [emberjs-atom](https://atom.io/packages/emberjs-atom) - Atom autocompletions, snippets for Ember.js.

Reviewer: ```suggestion
[emberjs-atom](https://atom.io/packages/emberjs-atom) - Atom autocompletions and snippets for Ember.js.
```

[atom-ember-snippets](https://github.com/mattmcmanus/atom-ember-snippets) - Ember.js ES6, Ember Data & Handlebars snippets for Atom editor.

[atom-ide-ember](https://github.com/josa42/atom-ide-ember) - Atom package to use the Ember Language Server.

## Sublime Text

A sophisticated text editor for code, markup and prose.

[ember-cli-sublime-snippets](https://github.com/terminalvelocity/ember-cli-sublime-snippets) - Ember CLI snippets for Sublime Text 3.
[ember-component-template-split-view](https://github.com/mmitchellgarcia/ember-component-template-split-view) - Super simple Sublime Text plugin that will let you open corresponding template or route files with EmberJS components.

## neovim

> You'll want to remove any linter / completion manager you currently have installed (or disable them for js/ts/hbs files), and follow the install guides for the following packages:

Reviewer: ```suggestion
You'll want to remove any linter / completion manager you currently have installed (or disable them for `js/ts/hbs` files), and follow the install guides for the following packages:
```

> [Conquer for Completion (coc) for nvim](https://github.com/neoclide/coc.nvim) - coc is an intellisense engine for neovim which takes control over all linting, hinting, and language-server integration.

Reviewer: ```suggestion
[Conquer for Completion (coc) for nvim](https://github.com/neoclide/coc.nvim) -  an `intellisense` engine for neovim which takes control over all linting, hinting, and language-server integration.
```

> [coc-ember](https://github.com/NullVoxPopuli/coc-ember) - Ember.js language server extension including useful for configuratio instructions.

Reviewer: ```suggestion
[coc-ember](https://github.com/NullVoxPopuli/coc-ember) - Ember.js language server extension including useful for configuration instructions.
```

> [Ember Language Server](https://github.com/lifeart/ember-language-server/tree/component-context-info) - Ember.js language server extension for coc.

Reviewer: ```suggestion
[Ember Language Server](https://github.com/lifeart/ember-language-server/tree/component-context-info) - Ember.js language server extension for `coc`.
```
